docs/en/how_to_setup_object_storage.md

```diff
@@ -71,7 +71,7 @@ The following table lists the object storage services supported by JuiceFS. Clic
 | [TiKV](#tikv)                                             | `tikv`     |
 | [Local disk](#local)                                      | `file`     |
 
-## S3 <span id='aws-s3'></span>
+## S3 {#aws-s3}
```

Comment:
It seems Github cannot parse the `{}` tag in Markdown

Reply:
It's ok when visit in https://juicefs.com/docs/